run.sh

```diff
@@ -272,6 +272,15 @@ workload_8_nodejs() {
                                                                                      --samples "${SAMPLES}"
 }
 
+workload_1_java() {
+  java -jar /usr/bin/java-cli/storage-shared-benchmarking-0.0.1-SNAPSHOT.jar -project="${PROJECT}" \
```

Comment:
You'll want to generalize the jar version with glob `*` because the version is set by dockerfile building of dependencies.

Reply:
I think we should be alright here since storage-shared-benchmarking is not being released thus the version shouldnt update.

Reply:
I'm assuming that you're using an internal Dockerfile to build this jar;
It's more of a debugging question; How will you be able to determine which version of java-storage and CLI implementation you're using?

Reply:
that is a good question, in essence we can't right now. we would have to know when we last deployed and what the state of the branch we are targeting was at that time.

Reply:
When we build the docker image, we should generate a manifest file where we can write the git sha, library versions etc in the event we need to refer back to an old image we have the necessary metadata to investigate.

It could be as simple as running the following when the image is built
```
git rev-parse --verify HEAD > /usr/lib/java-cli/java-storage-revision
mvn --projects storage-shared-benchmarking dependency:tree > /usr/lib/java-cli/storage-shared-benchmarking-dependencies
```

Reply:
this has been addressed in the docker PR, can this be merged now?

Reply:
Merge away!